static/js/compose.js

```diff
@@ -1296,6 +1297,12 @@ export function initialize() {
         clear_preview_area();
     });
 
+    // Expansion and Collapsing of the composebox from original height to full height
+    $("#compose").on("click", "#full_size_compose_button", (e) => {
+        e.preventDefault();
+        compose_height.toggle_compose_height();
+    });
```

Comment:
This might be done more cleanly by having different IDs for the toggle buttons in the two cases, and having each just call the appropriate function; it's harder to reason about code that does this sort of toggling logic rather than setting the state to what's desired.

Reply:
(Or by checking for the presence of an outer CSS class that controls the size of the compose box).

Reply:
I have made two different IDs for toggle buttons with different icons. This now looks better.

Reply:
Looks good.  What do you think about moving these functions to `compose_ui.js`, and having `compose_height.js` be a section in that module?  I think that'd result in more intuitive naming, and that's already where we have `compose_ui.autosize_textarea`.